tools/summarize-topic

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import urllib.parse
+
+from litellm import completion
+
+import zulip
+
+os.environ["HUGGINGFACE_API_KEY"] = "YOUR_API_KEY"
```

Comment:
Let's either read this from a new section in `~/.zuliprc` (preferred) or just put in the instructions to run it via `env HUGGINGFACE_API_KEY=key ./tools/summarize-topic`

Reply:
Added all the API keys from `~/.zuliprc`.